comp/core/gui/guiimpl/platform_nix.go

```diff
@@ -11,10 +11,22 @@ import (
 	"fmt"
 )
 
+const docURL = "https://docs.datadoghq.com/agent/basic_agent_usage/?tab=Linux"
```

Comment:
What do you think of linking to the GUI section directly ?
```suggestion
const docURL = "https://docs.datadoghq.com/agent/basic_agent_usage/?tab=Linux#datadog-agent-manager-gui"
```

Reply:
Good idea ! Made here 8ac186ced8a9eb43ab035923ff43cfadc11ca57f